test/fixtures/classic-to-flat/input.js

```diff
@@ -1,38 +1,78 @@
 module.exports = {
   app: {
     'app.js': '// app',
+
```

Comment:
I added 2 examples. The remaining changes---whitespaces, comments, and uses of `join()`---were to improve readability.